.github/actions/setup-builder/action.yaml

```diff
@@ -33,7 +33,18 @@ runs:
       shell: bash
       run: |
         apt-get update
-        apt-get install -y openjdk-${{inputs.jdk-version}}-jdk protobuf-compiler
+        apt-get install -y protobuf-compiler
+
+    - name: Install JDK 17
+      uses: actions/setup-java@v4
+      with:
+        distribution: 'adopt'
+        java-version: '17'
```

Comment:
Instead of `inputs.jdk-version`, this is fixed to 17 now?

Reply:
yea I should use `inputs.java-version` here